compiler/rustc_target/src/spec/riscv64gc_unknown_hermit.rs

```diff
@@ -0,0 +1,21 @@
+use crate::spec::Target;
+use crate::spec::{CodeModel, RelocModel, TlsModel};
+
+pub fn target() -> Target {
+    let mut base = super::hermit_base::opts();
+    base.cpu = "generic-rv64".to_string();
+    base.max_atomic_width = Some(64);
+    base.features = "+m,+a,+f,+d,+c".to_string();
+    base.code_model = Some(CodeModel::Medium);
+    base.relocation_model = RelocModel::Pic;
```

Comment:
This is already a default, no need to override it.

Reply:
Yes, i will remove the line.